src/test/resources/features/11-allergy/allergies.feature

```diff
@@ -5,14 +5,28 @@ Feature: Allergies Management
     And a user clicks on Allergies link
     Then the system loads Allergies page
 
-  @run
```

Comment:
Thanks @kdaud  for working on this, why are you removing @run annotation

Reply:
> why are you removing @run annotatio

  Ideally @run annotation should be implemented to a scenario when the step Definition is defined. Since the step definition for the new feature file has not been implemented then I think the annotation shouldn't be included